.github/workflows/goreleaser.yml

```
@@ -10,7 +10,7 @@ permissions:
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

```suggestion
    runs-on: ubuntu-latest
```

We use `ubuntu-latest` on all the other workflows so I'd tend to use it here as well for consistency, we also won't need to systematically update the version and use the latest lts.

---

@brad-allora bumping this ^

---

I'm not sure I agree on using `latest` here. Using latest will introduce potential variability, while using the same fixed version enforces reliability (won't introduce additional changes).

---

I agree with using a specific version, but we should update the other workflows to be consistent.